package.json

```diff
@@ -83,6 +83,7 @@
     "@types/react": "^16.9.11",
     "@types/react-dom": "^16.9.3",
     "@types/storybook__react": "^4.0.2",
+    "@types/styled-components": "^4.4.2",
```

Comment:
Should the package.json also include constraint/enforcment for using typescript 2.8?

Reply:
Specifically 2.8 and 2.8 only, or 2.8 or greater?

Reply:
I would think greater than 2.8, right @jasnross?

Reply:
It's definitely a good idea, but I'm not aware of a way to do so yet, although there is some discussion about it [here](https://github.com/microsoft/TypeScript/issues/32166). 

Do you know of an existing way to enforce the minimum TypeScript version required by consuming packages?

Reply:
I've been thinking about it and since typescript isn't peerDependency and grommet is installing it as part of its devDependencies we should be good with it as it.